Components/BlockCookieNotification/index.twig

```diff
@@ -0,0 +1,12 @@
+<div class="flyntComponent" is="flynt-block-cookie-notification">
+  <div class="notification">
+    <div class="notification-container">
+      <div class="notification-top">
+        {% include 'Partials/NotificationText/index.twig' %}
+      </div>
+      <div class="notification-bottom">
+        {% include 'Partials/AcceptButton/index.twig' %}
```

Comment:
I really don't like using Partials for these. They are just a single div and a single button. Just let them be here.